### PR TITLE
MBS-7669 Add filter troubleshooting docs

### PR DESCRIPTION
**Pull request by sergioboi**

### CODE REVIEW

File: docs/content/docs/test/Runner.md [hunk 1]
```diff
@@ -525,6 +525,26 @@ instrumentation {
 ./gradlew instrumentationLocal -PincludedAnnotation="package.AnotherAnnotationClassName" 
 ```
 
+## Find out how filters applied
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]:
```suggestion
## Find out how filters were applied
```

File: docs/content/docs/test/Runner.md [hunk 2]
```diff
@@ -525,6 +525,26 @@ instrumentation {
...
+
+If build finished successfully It will produce files in which you must find filters debug information
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]:
```suggestion
If build finished successfully there will be files with debug information
```
I think we have no reason for "must".

File: docs/content/docs/test/Runner.md [hunk 3]
```diff
@@ -525,6 +525,26 @@ instrumentation {
...
+Files will be located at: \
+ `<Project root folder>/outputs/<subproject name>/instrumentation/<instrumentation task name>/filter`
+
+### Find out what filter config was
+
+Look at file `filter-config.json`
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md:536]: Can we add some details about the content? Maybe a small example.

Comment (sergioboi) [reply docs/content/docs/test/Runner.md:536]: It's too simple. If we add some detail we must support those details. I don't want to support those files behavior

Comment (eugene-krivobokov) [reply docs/content/docs/test/Runner.md:536]: Ok

File: docs/content/docs/test/Runner.md [hunk 4]
```diff
@@ -525,6 +525,26 @@ instrumentation {
...
+
+### Find out what filters applied
+
+Look at file `filters-applied.json`
+
+### Find out what tests were filtered
+
+0. Look at file `filters-excludes.json`
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md:544]: Why do we need an ordered list? It contains only one element.

File: docs/content/docs/test/Runner.md [hunk 5]
```diff
@@ -525,6 +525,26 @@ instrumentation {
...
+- You may find filtered tests by the filter by the filter name declared in `filters-applied.json`
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md:545]: It's hard for me to understand the reference. Maybe we can simplify it.

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/content/docs/test/Runner.md [hunk 6]
```diff
@@ -525,6 +525,26 @@ instrumentation {
 ./gradlew instrumentationLocal -PincludedAnnotation="package.AnotherAnnotationClassName" 
 ```
 
+## Find out how filters were applied
+
+If build finished successfully there will be files with debug information
```
Comment (sergioboi) [inline docs/content/docs/test/Runner.md:530]:
```suggestion
If build finishes successfully It will produce files with debug information
```

**Pull request merged**